_hackbars/Arnab_ghosh.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: hackbar
+title: <First puu request>
+author: <ARNAB GHOSH>
+---
+
+<!-- Hi folks! this is my first step in open source-->
+
+---
+
+## <!-- Your MESSAGE on climate change -->
+
+![way towards retarding climate change]({{https://pixabay.com/}}/assets/images/<arnab_hackto-ber>.jpg)
```

Comment:
hey @ghosharnab969, you need to download the image and put on the `assets/images/` directory and link the image here in the `_hackbars/Arnab_ghosh.md` file.

For for more details, you can look at the documentation.

Reply:
ok sure!